app/models/authenticator.rb

```diff
@@ -191,7 +191,7 @@ def userid_for(_identity, username)
     end
 
     def authorize_queue?
-      !MiqEnvironment::Process.is_ui_worker_via_command_line?
+      !defined?(Rails::Server)
```

Comment:
Replaced MiqEnvironment::Process call with a Rails constant check (so we can delete MiqEnvironment::Process)